js/background.js

```diff
@@ -239,6 +240,13 @@
             view.onProgress(count);
         }
     }
+    function onSettings(ev) {
+        if (ev.settings.readReceipts) {
```

Comment:
This could be simplified with something like

`storage.put('read-receipt-setting', Boolean(ev.settings.readReceipts));`